packages/builder/src/components/integration/RestQueryViewer.svelte

```diff
@@ -362,6 +363,13 @@
       notifications.error("Error getting datasources")
     }
 
+    try {
+      // load the environment variables
+      await environment.loadVariables()
+    } catch (error) {
+      notifications.error("Error getting environment variables")
```

Comment:
Can we present a bit more of a useful error here? This should never happen, would be good to include some of the error message e.g. `Error getting environment variables - ${error}`.